source/commands/env/export/components/ExportContent.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useApiKeyApi } from '../../../../hooks/useApiKeyApi.js';
+import { useAuth } from '../../../../components/AuthProvider.js';
+import { ExportOptions } from '../types.js';
+import { ExportStatus } from './ExportStatus.js';
+import { useExport } from './hooks/useExport.js';
+import fs from 'node:fs/promises';
+
+export const ExportContent: React.FC<{ options: ExportOptions }> = ({
+	options: { key: apiKey, file },
+}) => {
+	const { validateApiKeyScope } = useApiKeyApi();
+	const { authToken } = useAuth();
+	const key = apiKey || authToken;
+	const { state, setState, exportConfig } = useExport(key);
+
+	React.useEffect(() => {
```

Comment:
Please use the convention of importing the function and use them without the `React.`

```
import React, { useEffect } from 'react';

...

useEffect(() => {
```